src/go/k8s/tests/e2e/console-admin-api/03-assert.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: call-console
+  namespace: console-admin-api
+status:
+  conditions:
+    - status: "True"
+      type: Complete
+  succeeded: 1
+
+---
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+- type: pod
+  selector: app.kubernetes.io/name=redpanda-console
+  tail: -1
```

Comment:
Maybe logs from Job pod could be beneficial to gather?

Reply:
I think this is ok since the job is just curl and it is consistent with other tests

Reply:
If job fails I doubt we will have any indication what happen.